packages/angular/src/generators/host/__snapshots__/host.spec.ts.snap

```diff
@@ -254,7 +254,12 @@ exports[`Host App Generator --ssr should generate the correct files 6`] = `
 exports[`Host App Generator --ssr should generate the correct files 7`] = `
 "const { withModuleFederationForSSR } = require('@nx/angular/module-federation');
 const config = require('./module-federation.config');
-module.exports = withModuleFederationForSSR(config);
+/**
+ * DTS Plugin is disabled in Nx Workspaces as Nx already provides Typing support Module Federation
```

Comment:
The wording is odd here, can we reword it? `Nx already provides Typing support Module Federation`